packages/rum-core/src/domain/rumEventsCollection/resource/resourceCollection.ts

```diff
@@ -37,7 +43,28 @@ export function startResourceCollection(
   sessionManager: RumSessionManager
 ) {
   lifeCycle.subscribe(LifeCycleEventType.REQUEST_COMPLETED, (request: RequestCompleteEvent) => {
-    lifeCycle.notify(LifeCycleEventType.RAW_RUM_EVENT_COLLECTED, processRequest(request, configuration, sessionManager))
+    if (request.type === RequestType.XHR || request.status === 0) {
```

Comment:
`request.status === 0` seems to resolve before we have time to set the `PerformanceObserver` so we need to capture this edge case